monGARS/core/conversation.py

```
@@ -554,23 +555,41 @@ async def generate_response(
+            try:
+                response_mapping = await llm_adapter.generate_response(  # type: ignore[attr-defined]
+                    prompt_bundle.text,
+                    task_type=task_type,
+                    response_hints=response_hints,
+                    formatted_prompt=prompt_bundle.chatml,
+                )
+            except LLMRuntimeError:
+                logger.exception("conversation.runtime.generate_failed")
+                raise
+            if not isinstance(response_mapping, Mapping):
+                raise TypeError("LLM integration must return a mapping")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)